src/Installer/finalizer/finalizer.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(SdkTargetFramework)-windows</TargetFramework>
+    <OutputType>WinExe</OutputType>
+    <Nullable>enable</Nullable>
+    <PublishAot Condition="$(NETCoreSdkPortableRuntimeIdentifier.StartsWith('win-'))">true</PublishAot>
+    <ControlFlowGuard>Guard</ControlFlowGuard>
+    <UseSystemResourceKeys>true</UseSystemResourceKeys>
+    <SatelliteResourceLanguages>en-US</SatelliteResourceLanguages>
+    <OptimizationPreference>Size</OptimizationPreference>
+    <StackTraceSupport>false</StackTraceSupport>
+    <UseCurrentRuntimeIdentifier>true</UseCurrentRuntimeIdentifier>
```

Comment:
`UseCurrentRuntimeIdentifier` pulls in the RID of the SDK. This doesn't work for the Win-x86 VMR leg. Instead, Arcade has tooling now to support automatically inferring RIDs for VMR legs.

Removing this property should fix the build.

```suggestion
```